traffic_ops/traffic_ops_golang/dbhelpers/db_helpers.go

```diff
@@ -1603,3 +1624,41 @@ func GetDSIDFromStaticDNSEntry(tx *sql.Tx, staticDNSEntryID int) (int, error) {
 	}
 	return dsID, nil
 }
+
+// AppendWhere appends 'extra' safely to the WHERE clause 'where'. What is
+// returned is guaranteed to be a valid WHERE clause (including a blank string).
```

Comment:
I think I was being a little cocky when I wrote this docstring - it's only guaranteed to be a valid WHERE clause when `where` is already a valid WHERE clause and `extra` is a valid extra condition statement.

Which is important to note, because it is always `AND`ed - you can't add `OR some = 'thing'` or it'll break. Although tests should probably catch that pretty fast.

I won't ask you to change the GoDoc, but it's a bit more optimistic than it should be.